docker/docker_entrypoint.sh

```diff
@@ -11,6 +18,7 @@ BASE_PATH=${BASE_PATH:?'Base path should be specified'}
 RPC_PORT=${RPC_PORT:-9933}
 WS_PORT=${WS_PORT:-9943}
 PORT=${PORT:-30333}
+EXTERNAL_PORT=${EXTERNAL_PORT:-${PORT}}
```

Comment:
I see that you are assigning `EXTERNAL_PORT` but where is it used?

Reply:
https://github.com/Cardinal-Cryptography/aleph-infra/blob/main/initContainer/entrypoint.sh this is where it's used. External port can be different than port variable while setting up a public-addr for testnet for example.